ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-commands.sh

```diff
@@ -4,11 +4,13 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+export KUBECONFIG=/var/run/kubeconfig/kubeconfig
 job-run-aggregator analyze-job-runs \
   --google-service-account-credential-file=${GOOGLE_SA_CREDENTIAL_FILE} \
   --job=${VERIFICATION_JOB_NAME} \
   --aggregation-id=${AGGREGATION_ID} \
   --explicit-gcs-prefix=${EXPLICIT_GCS_PREFIX} \
   --job-start-time=${JOB_START_TIME} \
```

Comment:
does this need `-query-source=cluster `?

Reply:
Yup, it is two lines below.